pkg/providers/aws/cluster_network_provider.go

```diff
@@ -28,25 +28,52 @@ import (
 
 //todo we need to ensure we test that if `IsEnabled` is true, it remains true for consecutive reconciles
 
+const (
+	tagName                = "Name"
```

Comment:
Might move this out to `tags.go` as it's a generic AWS tag for display names in the console